src/main/java/Command.java

```diff
@@ -1,3 +1,3 @@
 public enum Command {

```

Comment:
Good use of enumeration class to store the commands. Perhaps can add the enum as part of UI.java ?